mautic/mautic
# [UI] Dashboard widgets

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [X]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR increases the widget padding on the dashboard, removes its borders and makes light gray the dashboard background color. These changes aim to modernize the interface.

Before:

https://github.com/user-attachments/assets/8aabcf90-5a74-4b70-a7ad-0cdd443bf008


After:

https://github.com/user-attachments/assets/d4487f07-5d3f-4a1c-8cda-e002dbffbc20




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the dashboard
3. Check the widgets and interact
